### PR TITLE
Meetings CRUD: Update (Node)

### PR DESCRIPTION
**Pull request by Aaron-TheCreator**
this PR addresses issue #5 to add a directory with sample request, response and explanation for PUT (update) request to webex meetings API

### CODE REVIEW

File: meetings/update/update_meeting.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+ *         this directory on the command line:
+ * 
+ *         node ./update_meeting.js
+ * 
+ * Step 6: Profit. Get your app listed in the Webex App Hub!
+ *         https://apphub.webex.com/
+ * 
+ */
+
+ const https = require('https'); // https://nodejs.org/api/https.html
```
Comment (DavidLiedle) [inline meetings/update/update_meeting.js:30]: Let's remove that leading space

File: meetings/update/update_meeting.js [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+
+const myWebexDeveloperToken = 'REPLACE WITH API KEY';
+
+const body = JSON.stringify({
+   title: 'Incorporating Vibranium into Modern Electronics: Pt. 1', // String, Required | Meeting title. The title can be a maximum of 128 characters long.
+   agenda: 'This meeting`s agenda includes discussing plans to incorporate new, extremely conductive copper alloys which create their own electro-magnetic fields.', // example of one of many options to update
```
Comment (DavidLiedle) [inline meetings/update/update_meeting.js:37]: Let's swap the backtick and single quotes here so the meeting info has the more standard look

File: meetings/update/update_meeting.js [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+   password: 'REPLACE WITH MEETING PASSWORD',                  // String, Required
```
Comment (DavidLiedle) [inline meetings/update/update_meeting.js:38]: Let's move the definition of this up top next to the `myWebexDeveloperToken` const.

**Review by DavidLiedle — CHANGES_REQUESTED** [review]
Hey, @Aaron-TheCreator this is looking great! Left a few change requests.

**Review by adamweeks — APPROVED** [review]
Looks good to me! I see you've addressed @DavidLiedle's requests. Let's squash these commits into one so that they don't stick around after the merge. Using conventional-commit format I would expect the commit message to be `feat: add meeting update code sample`

File: meetings/update/update_meeting.js (hunk 2 above)
Comment (Aaron-TheCreator) [reply meetings/update/update_meeting.js]: ✅

File: meetings/update/update_meeting.js (hunk 1 above)
Comment (Aaron-TheCreator) [reply meetings/update/update_meeting.js]: ✅

**Review by DavidLiedle — APPROVED** [review]

**Pull request merged**